test/core/ChainManager.test.js

```diff
@@ -0,0 +1,131 @@
+import SocketService from '../../src/core/SocketService';
+import ChainManager from '../../src/core/ChainManager';
+import {
+  listAllChains,
+  listAllSnapshots,
+  getBlockNumber,
+  mineBlock
+} from '../../src/core/ChainRequest';
+
+jest.setTimeout(10000);
```

Comment:
this affects the global jest environment, so it should not go in a single test file, but in global test setup (see [this note](https://jestjs.io/docs/en/jest-object.html#jestsettimeouttimeout))